api/data_pipeline.py

```
@@ -407,9 +536,231 @@ def transform_documents_and_save_to_db(
+            
+            # Step 1: Run splitter manually
+            splitter = TextSplitter(**configs["text_splitter"])
+            logger.info("Running text splitter...")
+            split_docs = splitter(original_docs)
+            logger.info(f"Text splitter produced {len(split_docs)} chunks")
+            
+            # Step 2: Run embedder manually
+            embedder = get_embedder()
+            embedder_config = get_embedder_config()
+            batch_size = embedder_config.get("batch_size", 500)
+            embedder_transformer = ToEmbeddings(embedder=embedder, batch_size=batch_size)
```

Ruff (0.11.9)

Local variable `embedder_transformer` is assigned to but never used

Remove assignment to unused variable `embedder_transformer`

(F841)